service/src/main/kotlin/fi/espoo/evaka/pis/controllers/ParentshipController.kt

```diff
@@ -43,6 +48,7 @@ class ParentshipController(private val parentshipService: ParentshipService, pri
         with(body) {
             return jdbi.transaction {
                 parentshipService.createParentship(it, childId, headOfChildId, startDate, endDate)
+                    .also { asyncJobRunner.scheduleImmediateRun() }
```

Comment:
Isn't this `also` in the wrong place? Now it's called inside `transaction`

Reply:
:facepalm: need more :coffee: